source/getting-started/use-cases/configuration-assessment.rst

```diff
@@ -68,18 +68,21 @@ macOS SCA rule example:
 
 Below is an example of the results of a configuration assessment evaluation. These can be obtained through the web user interface or directly through the Wazuh RESTful API.
 
-.. thumbnail:: /images/getting-started/use-cases/wazuh-use-cases-sca01.png
-   :title: Security configuration assessment inventory dashboard
+.. thumbnail:: /images/getting-started/use-cases/security-configuration-assessment.png
```

Comment:
This the available  policies for the inventory

Reply:
Changes